### PR TITLE
Fix Vancian bills remain as chargeless items after ripping them

### LINKED ISSUE #68677: [Xedra Evolved] Vancian bills remain as chargeless items after ripping them
### Describe the bug

Vancian bills are described as being torn to activate them, but after using them to cast Rip they remain as items. Attempting to use them again will result in a message about it not having enough charges, but the name remains the same and charges aren't listed anywhere.

### Attach save file

N/A

### Steps to reproduce

1. Use a Vancian bill.
2. See that it remains, attempt to use it again to confirm it doesn't have enough charges.

### Expected behavior

Vancian bills should be destroyed after use, or otherwise show that they are spent and now useless.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3570 (22H2)
- Game Version: cfcc682 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Slowdown Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Blaze Industries [blazeindustries],
    Xedra Evolved [xedra_evolved],
    Mythical Martial Arts [MMA],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Tamable Wildlife [Tamable_Wildlife],
    Mind Over Matter [mindovermatter],
    Bombastic Perks [bombastic_perks],
    Portal Storms Ignore NPCs [personal_portal_storms]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Fix #68677

#### Describe the solution
Gives the spell a component for the rip_ticket I'll need to test this

#### Describe alternatives you've considered
No other potential options currently

#### Testing
Needs testing

### CODE REVIEW

File: data/mods/Xedra_Evolved/spells/item_spells.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
     "shape": "blast",
     "valid_targets": [ "hostile", "ground" ],
     "flags": [ "LOUD", "NO_LEGS", "RANDOM_DAMAGE" ],
+    "components": [ "rip_ticket" ],
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/spells/item_spells.json:81]:
```suggestion
    "components": [ "rip_ticket", 1 ],
```

**Comment by GuardianDll** [comment]
It won't work since bill is an item, and casting spell from item bypasses multiple spell fields, including requirements (not saying there is no rip_ticket for requirement added in this PR)
I propose straigh obsolete the vancian bill as it doesn't fit any class in the game now

**Comment by Maleclypse** [comment]
> It won't work since bill is an item, and casting spell from item bypasses multiple spell fields, including requirements (not saying there is no rip_ticket for requirement added in this PR) I propose straigh obsolete the vancian bill as it doesn't fit any class in the game now



> It won't work since bill is an item, and casting spell from item bypasses multiple spell fields, including requirements (not saying there is no rip_ticket for requirement added in this PR) I propose straigh obsolete the vancian bill as it doesn't fit any class in the game now


Ok  that makes sense.  I might make reflavor them slightly and drop from a crazed conspiracy theorist mob I've been considering making.

**Comment by Maleclypse** [comment]
Superceded by recent PR

**Pull request closed without merging**